### PR TITLE
IGNITE-10354 Failing client node due to not receiving metrics updates

### PR DESCRIPTION
**Pull request by gromtech**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java [hunk 1]
```diff
@@ -499,6 +499,13 @@ public void clientAliveTime(long aliveTime) {
         this.aliveCheckTime = U.currentTimeMillis() + aliveTime;
     }
 
+    /**
+     * @return Client alive check time.
+     */
+    public long getAliveCheckTime() {
```
Comment (alamar) [inline modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java]: By CC this method should be called aliveCheckTime()

File: modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java [hunk 2]
```diff
@@ -5273,6 +5273,11 @@ private void processMetricsUpdateMessage(TcpDiscoveryMetricsUpdateMessage msg) {
                             if (clientNodeIds.contains(clientNode.id()))
                                 clientNode.clientAliveTime(spi.clientFailureDetectionTimeout());
                             else {
+                                if (clientNode.getAliveCheckTime() == 0L) {
+                                    // Init client alive time.
```
Comment (alamar) [inline modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java]: I don't think this comment is needed.

File: modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java (hunk 1 above)
Comment (alamar) [reply modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java]: You should probably call it clientAliveTime() to match setter

Comment (gromtech) [reply modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java]: Agree. Sorry, forgot about this rule.

File: modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java (hunk 2 above)
Comment (gromtech) [reply modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java]: Agree.

**Pull request closed without merging**